### PR TITLE
Tut: Enable `collapse = TRUE`

### PR DESCRIPTION
**Pull request by GegznaV**
In the code of all tutorial `.Rmd` files in the PR, `knit_hook`:

```r
knitr::opts_chunk$set(collapse = FALSE)
knitr::knit_hooks$set(document = function(x){
  gsub("```\n*```r*\n*", "", x)
})
```

 is replaced with `collapse = TRUE` :

```r
knitr::opts_chunk$set(collapse = TRUE)
```
Solves #2388

### CODE REVIEW

**Comment by jakob-r** [comment]
Can you preview this pages section (e.g. with a screenshot):

https://mlr-org.github.io/mlr/articles/tune.html#performing-the-tuning

**Comment by GegznaV** [comment]
These print-screens are from the tutorial locally built on my computer:

image

image

image

@jakob-r, are the print-screens what you are interested in?

**Comment by GegznaV** [comment]
If you are searching for code fragmentation as in the example below, `collapse = TRUE` prevents from it.

image

**Comment by pat-s** [comment]
```
knitr::opts_chunk$set(collapse = FALSE)
knitr::knit_hooks$set(document = function(x){
  gsub("```\n*```r*\n*", "", x)
})
```

AFAIK these settings were important in the old `mkdocs` setups that we used before, weren' they @jakob-r?  
One reason was to not have a new output chunk for every code line as it shows up in your last screenshot. 

But apparently we have it there, but only in the "Get started" vignette. That's because we only have `collapse = FALSE` there while in all others we have the code above (probably just a copy/paste miss).

So we basically don't collapse the code by default and then manually remove all the empty lines between the code. Using `collapse = TRUE` does exactly this on a first glance. 

So in summary: This does not change anything to all the pages besides the "Get started" one because there we only used `collapse = FALSE` currently. 

But maybe I am overseeing an important point here from the old days why this was implemented this way. So I would wait for @jakob-r Feedback on this.

**Comment by jakob-r** [comment]
There had been issues with `collapse = TRUE` just working partially which made this workaround necessary at some point. So probably @pat-s is correct. Thanks for providing the sceens @GegznaV. I just wanted to make sure.

**Comment by GegznaV** [comment]
Closes #2388
